3-Authorization-II/web-api/resource-api/src/main/resources/authentication.properties

```
@@ -0,0 +1,22 @@
+# authority / tenant address is required to determine correct issuer (iss) claim
+# and to get correct signing keys for verification
+
+# client ID of the app is the audience (aud) claim required in the incoming tokens
+
+# scopes are required to determine if the incoming token has the required scopes to access your app
+aad.scopes=access_as_user
+
+########################################################################################
+#For V2 tokens, uncomment the two values (authority and client ID) in the section below:
+
+# aad.clientId=Enter_Your_Tenant_ID_Here
+# aad.authority=https://login.microsoftonline.com/Enter_Your_Tenant_ID_Here/
+
+########################################################################################
+#For V1 tokens, uncomment the two values (authority and client ID) in the section below:
+
+aad.clientId=api://Enter_Your_Client_ID_Here
+aad.authority=https://sts.windows.net/Enter_Your_Tenant_ID_Here/
```

```suggestion
aad.authority=https://sts.windows.net/Enter_Your_Tenant_ID_Here/
```

The readme instructs to set accessTokenAcceptedVersion to 2. So I think this section should be commented by default and the section above should be uncommented by default